docs/en/week14/14-3.md

```diff
@@ -94,20 +94,20 @@ Unlike $L_2$ weight decay, the $L_1$ regularization will "kill" components that
 Dropout involves setting a certain number of neurons to zero randomly during training. This prevents the network from learning a singular path from input to output. Similarly, due to the large parametrisation of neural networks, it is possible for the neural network to effectively memorize the input. However, with dropout, this is a lot more difficult since the input is being put into a different network each time since dropout effectively trains a infinite number of networks that are different each time. Hence, dropout can be a powerful way of controlling overfitting and being more robust against small variations in the input.
 
 <center>
-<img src="{{site.baseurl}}/images/week14/14-3/network_without_dropout.jpg" style="background-color:#DCDCDC;" /><br>
-Figure 2: Network without dropout
+<img src="{{site.baseurl}}/images/week14/14-3/network_without_dropout.JPG" style="background-color:#DCDCDC;" /><br>
```

Comment:
This is going to break the website. The file extension is `jpg`, not `JPG`.
I've already fixed this previously. Why are you undoing this? Did you try to run the website locally? Does it work?

Reply:
I didn't run it locally. But something is wrong with the current extention because the jpg images are not displayed correctly. It's not the case for the png images. 
If you have a look at the folder _\images\week14\14-3_ , the extension is JPG and not jpg so I guess that is the problem.